PokemonAPI/PokemonAPI/Controllers/HealthCheckController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace PokemonAPI.Controllers
+{
+    public class HealthCheckController : BaseController
+    {
+        [HttpGet]
+        public IActionResult Ping()
+        {
+            return Ok("Healthy");
+        }
+    }
```

Comment:
Just basic implementation of the controller to which we can access at all times to check if our system is Healthy